spec/lib/content_exporter_spec.rb

```diff
@@ -0,0 +1,199 @@
+locale_results_link_fixture = {
+  group_one: {
+    subgroup_one: {
+      title: "I am the title for Group one subgroup one",
+      items: [
+        {
+          id: "0001",
+          support_and_advice: false,
+          text: "This is a row that only has text, it will appaer like a paragraph",
+        },
+        {
+          id: "0002",
+          support_and_advice: false,
+          text: "This is a row has text and an href, it will appaer as an anchor tag",
+          href: "http://test.stubbed.gov.uk",
+        },
+        {
+          id: "0003",
+          support_and_advice: false,
+          text: "This is a row has text, an href and group criteira, it will appear as an anchor tag if the user's answers match the criteria",
+          href: "http://test.stubbed.llyw.cymru",
+          show_to_nations: %w[Wales],
+        },
+      ],
+      support_and_advice_items: [
+        {
+          id: "0004",
+          support_and_advice: false,
+          text: "This is a row has text, an href and multiple group criteira, it will appear as an anchor tag if the user's answers match the more complex criteria",
+          href: "http://test.stubbed.gb.gov.uk",
+          show_to_nations: %w[Wales Scotland England],
+        },
+      ],
+    },
+    subgroup_two: {
+      title: "I am the title for Group one subgroup two",
+      items: [{ id: "0005", text: "This is a row that only has text, it will appaer like a paragraph" }],
+    },
+  },
+  group_two: {
+    subgroup_one: {
+      title: "I am the title for Group two subgroup one",
+      items: [{ id: "0006", text: "This is a row that only has text, it will appaer like a paragraph" }],
+    },
+  },
+}
+
+locale_form_groups_fixture = {
```

Comment:
It'd be more maintainable to convert the results row fixture to a CSV and use that as the `csv_fixture`. We're not testing the 'csv' module here, so it's safe to use.

Reply:
Do you mean take this and turn like @177 into something like:
```
let(:csv_fixture) { CSV.generate { |csv| csv << locale_form_groups_fixture} }
```

Or something similar?
So one fixture instead of two?

Reply:
Humm played with a few options here,
Couldn't find something that made sense. Perhaps we can merge this in and follow up with a few maintainance tweaks in a PR once we've got content updating from these exports / imports?

Reply:
I was going for the one fixture over two, but looking into implementing it in more detail, the code overlap with the method is too large to retain the tests validity. 

It's probably best left as is 🙂